### ISSUE TITLE
Right click menu for better accessibility

### ISSUE THREAD

**Issue by MrPixelized** [vector-im/element-web#16480]
On wide-screen monitors, **deleting, replying to, or reacting to a message can be somewhat frustrating**, as you need to hover over a message and then move it all the way to the side of the screen to get to the right buttons. This placement for buttons is a bit slow and cumbersome, but it is also not baked into the typical computer user's way of thinking, so I have had friends who have **failed to find these buttons** prior to me telling them.

**Right click menus** are great. They are very fast, remove the need to aim or remember the location of specific UI elements, and are generally the first thing people think to try when the want to 'do something to a thing'. This makes them **more intuitive** for newcomers and **slightly faster** for people who have learned the software.

I have also considered having the buttons pop up at your cursor's location when you hover over the message, but that seem less elegant design-wise. The way the current buttons pop up is still a good way of hinting that features like message deletion and replies exist without looking off.

**Comment by LumiOfTheNosity** [comment]
I second this. :)

**Comment by SimonBrandner** [comment]
Related to [#16182](https://github.com/vector-im/element-web/issues/16182)

**Comment by waclaw66** [comment]
I'm really missing Discord user experience. At least the context menu on the message would be appreciated, as it works for rooms.
